index.js

```diff
@@ -5,10 +5,26 @@ console.log("Here's your header:", header)
 
 /***** Deliverable 2 *****/
 
+header.style.color = "brown"
 
 
 /***** Deliverable 3 *****/
-// console.log('This is what the traveler object looks like: ', traveler)
+console.log('This is what the traveler object looks like: ', traveler)
+//  *name*, *nickname*, and *photo* 
+
+const profile = document.querySelector('#profile')
+console.log(profile)
+const image = profile.querySelector('img')
+console.log(image)
+const name = profile.querySelector('h2')
+console.log(name)
+const nickname = profile.querySelector('em')
+console.log(nickname)
```

Comment:
Good work testing your code with those console logs to ensure that you're accessing the correct element! It's an effective approach and habit